czecs-resque.json

```diff
@@ -31,7 +31,7 @@
         },
         {
           "name": "QUEUE",
-          "value": "data_migration,q03_pipeline_run,transfer_basespace_files,generate_bulk_download,precache_report_info,*"
+          "value": "data_migration,q03_pipeline_run,result_monitor_loader,transfer_basespace_files,initiate_s3_cp,generate_bulk_download,precache_report_info,*"
```

Comment:
I'm taking a look on these queue priorities and I noticed this `data_migration`. @MarkAZhang Do we still need that queue? It looks it is only being used for `PopulateContigSpeciesTaxids` (https://github.com/chanzuckerberg/idseq-web/blob/62a010eca46fdffe7e2ffdc215ae18f9095b5c49/app/jobs/data_migrations/populate_contig_species_taxids.rb#L6) and I couldn't find any parts of our code sending messages to that queue.